app/views/categories/new.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="form-style mx-5">
+  <h1>New category</h1>
+  <%= render "form", category: @category %>
+
+  <br>
+
+  
+    <%= link_to "Back to categories", categories_path, class:"btn bg-blue mb-5" %>
+  
+</div>
```

Comment:
- [ ] As the previous reviewer suggested, please use colors font, colors, etc that were used in the [design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=). Please note that the form buttons for both categories and transactions should be `#5fb523`
- [ ] [OPTIONAL] Please consider placing the form buttons at the bottom of the page just like in the design
## Design Guideline
![image](https://user-images.githubusercontent.com/46839548/219826935-8e223b93-1400-45a1-9632-1d91b18b573c.png)

## Your Design
![image](https://user-images.githubusercontent.com/46839548/219826797-e2f8bd6b-6adf-433a-af8a-3f3905461f94.png)